runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/MonitoringUtil.java

```diff
@@ -180,14 +180,15 @@ public int compare(JobMessage o1, JobMessage o2) {
     return allMessages;
   }
 
-  public static String getJobMonitoringPageURL(String projectName, String jobId) {
+  public static String getJobMonitoringPageURL(String projectName, String regionId, String jobId) {
```

Comment:
Agreed with davor; Simultaneously, we make no backwards-compatibility guarantees about this package.